utils/chaos_ai/docker/requirements.txt

```diff
@@ -4,3 +4,4 @@ requests
 Flask==2.2.5
 Werkzeug==3.0.3
 flasgger==0.9.5
+kubernetes
```

Comment:
We might want to set the working version of kubernetes to make sure it doesn't break because of any changes in the upcoming versions. Thoughts?